package.json

```diff
@@ -6,6 +6,7 @@
         "@emotion/react": "^11.8.2",
         "@emotion/styled": "^11.8.1",
         "@hookform/resolvers": "^2.8.8",
+        "@mui/base": "^5.0.0-alpha.80",

```

Comment:
Is this a required dependency?

Reply:
I have imported now from mui/material, this is not required